src/redux/thunk/api.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import _ from 'lodash';
+import {
+  fetchContinent,
+  fetchContinentFailure,
+  fetchCountryDetails,
+} from '../actions/action';
+
+const baseURL = 'https://covid-api.mmediagroup.fr/v1/vaccines?continent=africa';
+
+export const fetchVacinated = () => (dispatch) => {
+  axios
+    .get(baseURL)
+    .then((res) => {
+      const reports = res.data;
+      const continent = [];
+      _.forEach(reports, (report) => continent.push(report));
+      dispatch(fetchContinent(continent));
+    })
+    .catch((err) => {
+      dispatch(fetchContinentFailure(err.message));
+    });
+};
+
+export const fetchCountry = (countryId) => (dispatch) => {
+  axios
+    .get(`https://covid-api.mmediagroup.fr/v1/vaccines?country=${countryId}`)
+    .then(() => {
+      dispatch(fetchCountryDetails(countryId));
+    });
+};
```

Comment:
- [ ] Make sure the project is running without including the data inside the `.env`

![t-3](https://user-images.githubusercontent.com/55635977/147164888-7837408f-be88-4cde-9de4-16d9bfbb9b9e.png)